README.md

```diff
@@ -177,6 +177,9 @@ services:
         arguments:
             # Pass the transport name used in config/packages/messenger.yaml
             $transportName: 'async'
+            # true enables partial SQS batch failure, see https://bref.sh/docs/function/handlers.html#partial-batch-response for more details.
+            # Enabling this without proper SQS config will consider all your messages successful
```

Comment:
As a reader, I'm not sure what to do next. How do I do that "proper SQS config"?

Correct me if I'm wrong but Lift automatically set that up, so we could probably mention that the user doesn't have anything to do if they deploy the README example? Not sure how to word that best though.

Reply:
I didn't wanted to add a lot of stuff to the docs in the repo and link to the bref docs that explains it already.

To be honest, I want to revamp the docs on this repo, it feels bloated and confusing to me. Some things are outdated too. Also, most config is boilerplate code that could maybe be done by a bundle or flex recipe. But that's another topic :sweat_smile: 

For the time being, I'll see how I can make that clearer and avoid blocking this PR.

Reply:
Ohhh right I completely missed the link!

Maybe make the link more visible?

Reply:
I've changed it a bit, hope it's clearer.